src/runtime/go.mod

```diff
@@ -27,7 +27,7 @@ require (
 	github.com/gogo/protobuf v1.3.2
 	github.com/hashicorp/go-multierror v1.0.0
 	github.com/intel-go/cpuid v0.0.0-20210602155658-5747e5cec0d9
-	github.com/kata-containers/govmm v0.0.0-20210909155007-1b60b536f3c7
+	github.com/kata-containers/govmm v0.0.0-20211118155643-d432e21d6fc0
```

Comment:
can we use the latest version of govmm `aaac742` ?

Reply:
Updated to `aaac742`